docs/data-modeling/partials/classic-connectors/mongoDB/_command-create-tutorial.mdx

```diff
@@ -0,0 +1,84 @@
+The MongoDB data connector defines custom commands via
+[native mutations](/connectors/mongodb/native-operations/native-mutations.mdx).
+
+Within your connector's directory, you can add a new JSON configuration file to define a native mutation.
+
+```sh title="Create a new directory to store your native mutations:"
+mkdir -p app/connector/my_mongo/native_mutations/
+```
+
+```json title="Create a new file in your connector's directory:"
+// native_mutations/insert_user.json
+{
+  "name": "insertUser",
+  "description": "Inserts a user record into the database",
+
+  "arguments": {
+    "name": { "type": { "scalar": "string" } }
+  },
+
+  "resultType": {
+    "object": "InsertUser"
+  },
+  "objectTypes": {
+    "InsertUser": {
+      "fields": {
+        "ok": { "type": { "scalar": "double" } },
+        "n": { "type": { "scalar": "int" } }
```

Comment:
As a user, I'm confused by these field names. Is there something I'm missing, or can we change the names to make the keys clearer?

Reply:
I _think_ these come from the data connector and are default-ish. Can you confirm please @hallettj 🙏  Possible to change these?